front/src/routes/dashboard/style.css

```
@@ -655,6 +655,43 @@
+:global(.glass-theme .loader)::after {
+  width: 1.625rem;
+  height: 1.625rem;
+  margin: -0.8125rem 0 0 -0.8125rem;
+  border-width: 2px;
+}
+
+:global(.glass-theme .card.loading-border) {
+  animation: glassLoadingBorderBlink 2s infinite;
+}
+
+@keyframes glassLoadingBorderBlink {
```

Stylelint (17.14.0)

[error] Expected keyframe name "glassLoadingBorderBlink" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)